packages/stark-ui/src/util/dom.util.ts

```diff
@@ -97,4 +97,12 @@ export class StarkDOMUtil {
 	public static hasClass(element: Element, classname: string): boolean {
 		return element.classList.contains(classname);
 	}
+
+	/**
+	 * Check if passed input is equal to "true", empty string or true.
+	 * @param input - Input to be checked
+	 */
+	public static isInputEnabled(input?: string): boolean {
```

Comment:
I think this method should be put in a separate Util class instead of the DOMUtil because it is not related to the DOM at all. Otherwise we are mixing things and making our API a bit confusing.